superset/jinja_context.py

```diff
@@ -209,6 +205,7 @@ def __init__(
             "form_data": {},
         }
         self.context.update(kwargs)
+        BASE_CONTEXT.update(current_app.config["JINJA_CONTEXT_ADDONS"])
```

Comment:
This might be excessive, but I would perhaps break this out into an `init_base_context()` which does the above `if globals().get("BASE_CONTEXT") is None`.

Reply:
ok

Reply:
What about moving the initialization of the base context into the AppInitializer? You could then expose the base context via extensions.py